compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUSplitReduction.cpp

```diff
@@ -154,47 +155,45 @@ LogicalResult splitReductionImpl(Operation *op, int64_t size,
 class LLVMCPUSplitReductionPass
     : public LLVMCPUSplitReductionBase<LLVMCPUSplitReductionPass> {
  public:
-  LLVMCPUSplitReductionPass(bool fpReductionReordering)
-      : fpReductionReordering(fpReductionReordering) {}
+  LLVMCPUSplitReductionPass(bool fpReductionReordering) {
+    this->enableFpReductionReordering = fpReductionReordering;
```

Comment:
is it possible to initialize the super class with the bool flag here or there is no constructor for it?

Reply:
No, it's not. I think it can be initialized with table-gened OptionStruct, but no one is using it in IREE and MLIR, AFAIK.